crates/router/src/connector/forte/transformers.rs

```diff
@@ -101,9 +101,10 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for FortePaymentsRequest {
                     card,
                 })
             }
-            _ => Err(errors::ConnectorError::NotImplemented(
-                "Payment Method".to_string(),
-            ))?,
+            _ => Err(errors::ConnectorError::NotSupported {
```

Comment:
Instead of default handling('_'), can you expand all the cases and assign it to Error

Reply:
Updated!